snapshot.md

```diff
@@ -3,47 +3,165 @@
+
+Size in bytes: `{{ site.data.rolling_snapshot.filesize_bytes }}`
+
+Checksum (SHA256):
+
+```
+{{ site.data.rolling_snapshot.sha256 }}
+```
+
+[Artifact Metadata]({{ domain_name }}/rolling-snapshot-metadata?{{ randomNumber }})
 
-## Full snapshot
+## Archive tarball
 
-[Full Snapshot](${snapshot_name}.full)
+[Download Archive Tarball]({{ domain_name }}/{{ site.data.archive_tarball.filename }})
 
-HOW TO USE
+Block height: `{{ site.data.archive_tarball.block_height }}`
 
+Block hash: `{{ site.data.archive_tarball.block_hash }}`
+
+[Verify on TzStats](https://{{ tzstats_subdomain }}tzstats.com/{{ site.data.archive_tarball.block_hash }}){:target="\_blank"} - [Verify on TzKT](https://{{ tzkt_subdomain }}tzkt.io/{{ site.data.archive_tarball.block_hash }}){:target="\_blank"}
+
+Block timestamp: `{{ site.data.archive_tarball.block_timestamp }}`
+
+Size in bytes: `{{ site.data.archive_tarball.filesize_bytes }}`
+
+Checksum (SHA256):
+
+```
+{{ site.data.archive_tarball.sha256 }}
 ```
-wget https://delphinet.xtz-shots.io/tezos-delphinet-83316.rolling
-tezos-node snapshot import tezos-delphinet-83316.rolling --block BMDKHftB5biRp9ZdkcMaoGby5UhT38eR8ntB1SWQ54fDNqKgr1n
-```
\ No newline at end of file
+
+[Artifact Metadata]({{ domain_name }}/archive-tarball-metadata?{{ randomNumber }})
+
+## Rolling tarball
+
+[Download Rolling Tarball]({{ domain_name }}/{{ site.data.rolling_tarball.filename }})
+
+Block height: `{{ site.data.rolling_tarball.block_height }}`
+
+Block hash: `{{ site.data.rolling_tarball.block_hash }}`
+
+[Verify on TzStats](https://{{ tzstats_subdomain }}tzstats.com/{{ site.data.rolling_tarball.block_hash }}){:target="\_blank"} - [Verify on TzKT](https://{{ tzkt_subdomain }}tzkt.io/{{ site.data.rolling_tarball.block_hash }}){:target="\_blank"}
+
+Block timestamp: `{{ site.data.rolling_tarball.block_timestamp }}`
+
+Size in bytes: `{{ site.data.rolling_tarball.filesize_bytes }}`
+
+Checksum (SHA256):
+
+```
+{{ site.data.rolling_tarball.sha256 }}
+```
+
+[Artifact Metadata]({{ domain_name }}/rolling-tarball-metadata?{{ randomNumber }})
+
+## How to use
+
+### Archive Tarball
+
+Issue the following commands:
+
+```bash
+curl -L "{{ domain_name }}/{{ site.data.archive_tarball.filename }}" \
+| lz4 -d | tar -x -C "/var/tezos"
+```
+
+Or simply use the permalink:
+
+```bash
+curl -L "{{ domain_name }}/archive-tarball" \
+| lz4 -d | tar -x -C "/var/tezos"
+```
+
+### Rolling Tarball
+
+Issue the following commands:
+
+```bash
+curl -L "{{ domain_name }}/{{ site.data.rolling_tarball.filename }}" \
+| lz4 -d | tar -x -C "/var/tezos"
+```
+
+Or simply use the permalink:
+
+```bash
+curl -L "{{ domain_name }}/rolling-tarball" \
+| lz4 -d | tar -x -C "/var/tezos"
+```
+
+### Rolling Snapshot
+
+Issue the following commands:
+
+```bash
+wget {{ domain_name }}/{{ site.data.rolling_snapshot.filename }}
+tezos-node snapshot import {{ site.data.rolling_snapshot.filename }} --block ${BLOCK_HASH}
```

Comment:
I forgot to templatize block hash here. @orcutt989 will you be making any more changes in this file? Otherwise i will fix.

Reply:
Great catch will fix thanks.